src/components/status-card/status-card.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright contributors to the ibm-storage-odf-console project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as React from 'react';
+import * as _ from 'lodash';
+import {
+  Gallery,
+  GalleryItem,
+} from "@patternfly/react-core";
+import {
+  useK8sWatchResource,
+} from "@console/dynamic-plugin-sdk/api";
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+  HealthItem,
+  //AlertsBody,
+  //AlertItem,
+} from "@console/dynamic-plugin-sdk/provisional";
+import { 
+  getFlashsystemHealthState, 
+  //filterIBMFlashSystemAlerts,
+  //alertURL,
+ } from './utils';
+import { StorageInstanceKind } from '../../types';
+import {GetFlashSystemResource} from '../../constants/resources'
+
+/*
+export const IBMFlashSystemAlerts = withDashboardResources(
```

Comment:
We can create an issue for this and when API is ready we can push these changes or we can leave todos in code as well.
Commenting code is not a good practice, also misses the chance to update later.

Reply:
+1. You can push the uncommented version to a different branch in this repository. Then you'll only need to rebase and merge in the future, and we can also remove the comments here safely.